services/s3/src/main/java/software/amazon/awssdk/services/s3/presigner/S3Presigner.java

```diff
@@ -512,6 +513,8 @@ default PresignedAbortMultipartUploadRequest presignAbortMultipartUpload(
     @SdkPublicApi
     @NotThreadSafe
     interface Builder extends SdkPresigner.Builder {
+        Builder serviceConfiguration(S3Configuration serviceConfiguration);
+
```

Comment:
Can we include the javadoc here as well? I see you added it to the concrete implementation already.

Reply:
sure thing!